### ISSUE TITLE
SSR: Scrollbar width is not correctly

### ISSUE THREAD

**Issue by rossanmol** [MurhafSousli/ngx-scrollbar#64]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?
All (Windows 10)

### Versions

@angular/cdk: 6.4.6
@angular/common: 6.1.6
@angular/compiler: 6.1.6
@angular/core: 6.1.6
@angular/flex-layout: 6.0.0-beta.17
@angular/forms: 6.1.6
@angular/http: 6.1.6
@angular/material: 6.4.6
@angular/platform-browser: 6.1.6
@angular/platform-browser-dynamic: 6.1.6
@angular/platform-server: 6.1.6
@angular/router: 6.1.6
@angular/cli: 6.1.5
@angular/compiler-cli: 6.1.6
@angular/language-service: 6.1.6

### Repro steps
1. Add **ngScrollbar** element to a component
2. Enable server side rendering for application
3. Notice the view returned by server side renderer, before the client kicks in:
4. Native scrollbars are shown.

### The log given by the failure
No log errors, however it was noticed that the following element styled are applied to **ng-scrollbar-view**:
```scss
    width: calc(100% + NaNpx);
    height: calc(100% + NaNpx);
```

### Desired functionality
Native scrollbars should never show.

### Mention any other details that might be useful
Scrollbar returned by SSR rendered appication:
![8e016ad02a116663bf7adb2891f729f5](https://user-images.githubusercontent.com/18351988/46243266-a3980e80-c3d2-11e8-9023-8d252fb72a7a.png)

**Comment by MurhafSousli** [comment]
Hi @rossanmol the plugin gets the browser scrollbar width so it can hides them


https://github.com/MurhafSousli/ngx-scrollbar/blob/9ccc514d3e88fa99985155c1b462992923edc5d5/projects/ngx-scrollbar/src/lib/scrollbar.component.ts#L360-L375

It seems that in SSR it is returning `NaN`, did you try it on other browsers?

**Comment by rossanmol** [comment]
Hi @MurhafSousli, the following won't wokr on server side, as it has no understanding of `offsetWidth` and `clientWidth` .

```ts
const sw = element.offsetWidth - element.clientWidth;
```

I will try uploading a reproduction repo for this issue later today.

**Comment by MurhafSousli** [comment]
It should be fixed in v2.3.0, can you confirm that?
